backend/interview_questions/prompts.py

```diff
@@ -0,0 +1,22 @@
+INTERVIEW_QUESTION_PROMPT = (
```

Comment:
Since you are returning multiple interview questions, make the variable plural.
```suggestion
INTERVIEW_QUESTIONS_PROMPT = (
```